internal/build/job/maker.go

```diff
@@ -113,14 +122,48 @@ func (m *maker) MakeJob(mod ootov1alpha1.Module, buildConfig *ootov1alpha1.Build
 	return job, nil
 }
 
-func MakeSecretVolumes(secretRefs []v1.LocalObjectReference) []v1.Volume {
+func (m *maker) makeImagePullSecretVolume(secretRef v1.LocalObjectReference) v1.Volume {
+
+	vol := v1.Volume{
+		Name: volumeNameFromSecretRef(secretRef),
+		VolumeSource: v1.VolumeSource{
+			Secret: &v1.SecretVolumeSource{
+				SecretName: secretRef.Name,
+				Items: []v1.KeyToPath{
+					{
+						Key:  v1.DockerConfigJsonKey,
+						Path: "config.json",
+					},
+				},
+			},
+		},
+	}
+
+	return vol
```

Comment:
```suggestion
func (m *maker) makeImagePullSecretVolume(secretRef v1.LocalObjectReference) v1.Volume {
	return v1.Volume{
		Name: volumeNameFromSecretRef(secretRef),
		VolumeSource: v1.VolumeSource{
			Secret: &v1.SecretVolumeSource{
				SecretName: secretRef.Name,
				Items: []v1.KeyToPath{
					{
						Key:  v1.DockerConfigJsonKey,
						Path: "config.json",
					},
				},
			},
		},
	}
```

Reply:
Also, I don't think this should be a method.

Reply:
Agree. Fixing.